packages/reporter/src/commands/command.tsx

```diff
@@ -198,6 +199,7 @@ class Command extends Component<Props> {
             </span>
           </div>
         </FlashOnClick>
+        { model.state === 'pending' && !!timeRemaining && <div className='command-progress'><span style={{ animationDuration: `${timeRemaining}ms`, width: `${timeRemaining / model.timeout * 100}%` }} /></div> }
```

Comment:
Can we pull this out into something?

A function at a minimum but also appropriate would be a component since its state and styles are pretty self-contained.